src/globals/scss/objects/_form-group.scss

```diff
@@ -0,0 +1,23 @@
+@include exports("form-group") {
+
+  .govuk-o-form-group {
+    @include govuk-responsive-margin($govuk-spacing-responsive-6, "bottom");
+
+    .govuk-o-form-group:last-of-type,
+    .govuk-c-input:last-of-type,
+    .govuk-c-select:last-of-type {
```

Comment:
This seems brittle and incomplete

Is there a generic selector we can use here?

Reply:
This was a good call - the bottom margins on input and select are no longer needed as they are no wrapped in form-group that handles the margins 👍